backend/common/exceptions.py

```diff
@@ -140,6 +140,10 @@ class InvalidMpLink(commands.CommandError):
 
     pass
 
+class pastNow(commands.CommandError):
```

Comment:
The class name should start with a capital letter. Also, `pastNow` more feels like now has been passed (aka future), than the time is in the past, maybe you should rename it to something like `TimeInThePast` ?

Reply:
Chose TimeInThePast as suggested.